src/main/jte/bookings/create/conflict.jte

```diff
@@ -64,15 +64,15 @@
                             <p class="list-col-wrap">
                                 ${booking.getDescription().isBlank() ? context.lookup("bookings.description.none") : booking.getDescription()}
                             </p>
-                            <a class="btn btn-square btn-ghost" href="/${booking.getRoom().getId()}/bookings/${booking.getId()}" target="_blank" aria-label="${context.localize("bookings.view")}">
+                            <a class="btn btn-square btn-ghost" href="/${booking.getRoom().getId()}/bookings/${booking.getId()}" target="_blank" aria-label="${context.localize("bookings.view")}" title="${context.localize("bookings.view")}">
                                 <i class="fa-solid fa-eye"></i>
                             </a>
                         </li>
                     @endfor
                 @endif
             </ul>
             @template.util.form(csrf = csrf, action = "/" + attemptedBooking.getRoom().getId() + "/bookings/new/conflict", method = POST, content = @`
-                <button class="btn btn-primary" type="submit">${context.localize("bookings.conflict.solve")}</button>
+                <button class="btn btn-primary" type="submit" aria-label="${context.localize("bookings.conflict.tooltip.solve")}" title="${context.localize("bookings.conflict.tooltip.solve")}">${context.localize("bookings.conflict.solve")}</button>
```

Comment:
A better name might be `bookings.conflict.solve.tooltip` or `bookings.conflict.solve.description`.
Additionally, this should be set either for aria-label or for title, not for both